src/components/templates/Auditorium/components/Section/Section.scss

```diff
@@ -0,0 +1,105 @@
+@import "scss/constants";
+
+// If you change this, make sure to also change it in constants.ts
+$seat-size: 4vh;
+$seat-size-min: 25px;
+$seat-spacing: 3px;
+
+$spacing: calc(1 * min(var(--seat-size), var(--seat-size-min)));
+
+.section {
+  --seat-size: #{$seat-size};
+  --seat-size-min: #{$seat-size-min};
+  --seat-spacing: #{$seat-spacing};
+
+  background-attachment: fixed;
+  background-position: center 46%;
+
+  position: relative;
+  overflow: auto;
+  height: 100%;
+  padding: $spacing;
+  min-width: fit-content;
+
+  &__video-overlay {
+    position: absolute;
+
+    width: 100%;
+    height: 100%;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    pointer-events: none;
+  }
+
+  &__video {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    pointer-events: all;
+
+    width: 100%;
+    height: 100%;
+    border: 1px black;
+    border-radius: 15px;
+    box-shadow: 2px 2px 20px $white;
+  }
```

Comment:
We should use a generic name for this rather than `video` that implies 'main embeddable content' (and it should be the same name used for this concept in the couple of other places I have mentioned this pattern)

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1251/commits/c7e60d97eac7ff85f121f5b5d8a480f8f16c3d70